src/components/AttachmentModal.js

```diff
@@ -197,34 +262,19 @@ class AttachmentModal extends PureComponent {
                         />
                     )}
                 </Modal>
+
                 <ConfirmModal
-                    title={this.props.translate('attachmentPicker.attachmentTooLarge')}
+                    title={this.state.attachmentInvalidReasonTitle}
```

Comment:
This change causes a regression https://github.com/Expensify/App/issues/22665. We just reverted back the logic i.e store `translationKey` in the state instead of value. Translated while passing